### PR TITLE
AYR-1304 - Expand the list of textract formats

### PR DESCRIPTION
**Pull request by ltrasca**
## Changes in this PR
- Added more file formats that textract can extract text from
- Added tests to cover text related formats

## JIRA ticket
https://national-archives.atlassian.net/browse/AYR-1304

## Screenshots of UI changes

N/A

- [ ] Requires env variable(s) to be updated

### CODE REVIEW

**Review by anthonyhashemi — APPROVED** [review]
Nice!

File: data_management/opensearch_indexer/opensearch_indexer/text_extraction.py [hunk 1]
```diff
@@ -7,11 +7,40 @@
 logger = logging.getLogger()
 logger.setLevel(logging.INFO)
 
+SUPPORTED_TEXTRACT_FORMATS = [
```
Comment (anthonyhashemi) [inline data_management/opensearch_indexer/opensearch_indexer/text_extraction.py:10]: so is this all the formats it supports?

Comment (anthonyhashemi) [reply data_management/opensearch_indexer/opensearch_indexer/text_extraction.py:10]: also which ones do we not add tests for?

Comment (ltrasca) [reply data_management/opensearch_indexer/opensearch_indexer/text_extraction.py:10]: @anthonyhashemi yes should be all of them, taken from this list

image

At the moment tests dont cover the ones that require system dependencies like Tesseract or antiword

Comment (anthonyhashemi) [reply data_management/opensearch_indexer/opensearch_indexer/text_extraction.py:10]: Can you create a ticket for writing e2e tests for those please. Not to do now. We need to think of the best way to run e2e tests for the indexing.

Comment (ltrasca) [reply data_management/opensearch_indexer/opensearch_indexer/text_extraction.py:10]: @anthonyhashemi sure, created a ticket

**Pull request merged**